tests/rgw/sanity_rgw_multisite.py

```diff
@@ -92,7 +92,10 @@ def run(**kw):
         set_test_env(config, primary_rgw_node)
         set_test_env(config, secondary_rgw_node)
 
-        if primary_cluster.rhcs_version.version[0] == 5:
+        out, _ = primary_rgw_node.exec_command(
+            sudo=True, cmd="rpm -qa | grep ceph-common"
+        )
+        if int(out.split("ceph-common-")[-1].split(".")[0]) > 14 :
```

Comment:
I believe you are adding support for RHCS 6.0 and I believe it is better to do
```
if primary_cluster.rhcs_version.version[0] >= 5:
```

Reply:
Made changes as discussed